src/Footer.tsx

```diff
@@ -0,0 +1,134 @@
+// need to convert to typescript
```

Comment:
Please convert this to a `TODO` and add a corresponding issue link.
(i.e. `// TODO: need to convert to typescript (https://github.com/LetsMesh/Site/issues/42069)`)

Reply:
Oh my bad those were old comments that i forgot to delete

Reply:
So we don't want this written in TypeScript? I prefer it over JavaScript, but I'll let Ryan chime in on what's better. Ideally we should have all JavaScript or all TypeScript imo.

Reply:
So that was a comment left by the person who had this task previously where he did it in JS, I overwrote the file into typescript but I forgot to remove his comment, should be gone now

Reply:
Ah okay gotcha. I'm not used to React at all.